app/components/candidate_interface/equality_and_diversity_review_component.html.erb

```diff
@@ -0,0 +1 @@
+<%= render(SummaryCardComponent.new(rows: equality_and_diversity_rows, border: false, editable: @editable)) %>
```

Comment:
`border: false` 🤔 is that the design? If yes ignore me 👍

Reply:
Yush, also looks weird atm because we only have one field.

![image](https://user-images.githubusercontent.com/42817036/75233353-a1e19e80-57b0-11ea-9d25-f5624544fd6e.png)